ui/src/landing/CreateWish.js

```diff
@@ -3,40 +3,52 @@ import { Landing } from './splashScreen'
 import Childinfo from '../childinfo/ChildInfo'
 
 export default class CreateWish extends Component {
-
   constructor(props) {
     super(props)
     this.state = {
       name: '',
       age: '2',
-      wishType: ''
+      wishType: '',
+      showChildInfo: false
     }
   }
 
-  validFields = (age) => ({
-      validAge: age !== '' && (Number(age) < 18 && Number(age) >= 2)
+  validFields = age => ({
+    validAge: age !== '' && (Number(age) < 18 && Number(age) >= 2)
   })
 
   updateField = (field, value) => {
     this.setState({ [field]: value })
   }
 
-  selectWishType = (wishType) => {
-    this.setState({ wishType })
+  selectWishType = wishType => {
+    const { name, age } = this.state
+
+    if (name !== '' && age !== '') {
+      this.setState({ wishType, showChildInfo: true })
```

Comment:
Looks like this path is missed in the tests according to CodeCov, potentially modifying the test to use `selectWishType` directly vs. modifying the `showChildInfo` variable should resolve it. 

Let me know what you think.

Reply:
Thanks. I have made the change and pushed.